<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>392</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>OWLprog</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <underline>true</underline>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>СТАРТ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="yes_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <underline>true</underline>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>ДА</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="no_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <underline>true</underline>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>НЕТ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="result_show">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="help_res">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Объяснение решения</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ЕЩЁ</string>
    </property>
    <addaction name="reset"/>
    <addaction name="base"/>
    <addaction name="help"/>
    <addaction name="close"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="reset">
   <property name="text">
    <string>Начать сначала</string>
   </property>
  </action>
  <action name="help">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="close">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="base">
   <property name="text">
    <string>База знаний</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
